File: Lab1_0/C#/main.cs
using System;

class StringClassMain
{
    static void Main()
    {
        Console.Write("Enter a string: ");
        string input = Console.ReadLine();
        StringClass s = new StringClass(input);
        Console.WriteLine(s.text);
        Console.WriteLine(s.Length());
        s.ShiftRight();
        Console.WriteLine(s.text);
    }
}

File: Lab1_0/C++/main.cpp
#include <iostream>
#include "Class.h"

int main()
{
    std::string input;
    std::cout << "Enter a string line: ";
    std::cin >> input;

    StringClass s(input);
    std::cout << s.text << std::endl;
    std::cout << s.length() << std::endl;

    s.shiftRight();
    std::cout << s.text << std::endl;

    return 0;
}